README.md

```diff
@@ -156,9 +156,11 @@ Options:
   --write-kwargs DICT  Keyword arguments to pass to ase.io.write when saving
                        results. Must be passed as a dictionary wrapped in
                        quotes, e.g. "{'key' : value}".  [default: "{}"]
-  --log PATH           Path to save logs to.  [default: singlepoint.log]
-  --summary PATH       Path to save summary of inputs and start/end time.
-                       [default: singlepoint_summary.yml]
+  --log PATH           Path to save logs to. Default is inferred from the name
+                       of the structure file.
+  --summary PATH       Path to save summary of inputs, start/end time, and
+                       carbon emissions. Default is inferred from the name of
+                       the structure file.
```

Comment:
Don't these now also infer from the ensemble as well when md is done, ```NaCl-npt-T300.0-p1.0-log.yml```, and not in e.g. pure geomopt, ```H20-log.yml```. Do we need to comment on that here?

Obviously would effect a lot of docstrings in the other diff's so I won't comment individually

Reply:
Technically not until #279 is fixed, but yes it should do, good point. I think the docstrings for the MD CLI are the only one that needs changing, or am I missing something?

Reply:
Ah ok yes I was slightly confused by what's coming and when. 

And I think you are right it should just be for the md cli for this comment alongside the readme, I think as that is the only time we get the ```npt-TXXX-pXXX``` etc